src/main/java/de/caritas/cob/agencyservice/config/SecurityConfig.java

```diff
@@ -76,6 +87,21 @@ protected void configure(HttpSecurity http) throws Exception {
         .anyRequest().denyAll();
   }
 
+  @Bean
+  TenantResolver tenantResolver() {
+    return new TenantResolver(subdomainExtractor(), tenantControllerApi());
+  }
+
+  @Bean
+  SubdomainExtractor subdomainExtractor() {
+    return new SubdomainExtractor();
+  }
+
+  @Bean
+  de.caritas.cob.agencyservice.tenantservice.generated.web.TenantControllerApi tenantControllerApi() {
```

Comment:
please change to non-fully qualified import

Reply:
removed